azure/batch/scripts/predict_on_catalog.py

```diff
@@ -273,6 +342,7 @@ def odds_answer_below_bounds(predictions: np.ndarray,  question_indices: List[in
     """
     Calculate the predicted odds that the galaxy would have an infinite-volunteer vote fraction no higher than `bound'
     (for a given question and answer)
+
     e.g. the predicted odds that an infinite number of volunteers would answer `smooth' to `smooth or featured' less than 20% of the time
```

Comment:
should this comment update with the use of `featured-or-disk` question to align our use case?